### PR TITLE
Fix a few issues and styles

### PR DESCRIPTION
**Pull request by jazeee**
Fix Overview tab, Skill mod column is too narrow for double digits. #62

Fix Consider making grid columns use `word-break: break-word` #61

Fix Small formatting issues #60

* Innate spells, update column title to `Damage/Duration`.
* For arcane spells, adjust columns:
  * the mod column should not stretc
  * the target column should stretch.
  * Also, adjusted range and target to be stretch.
* Fix button hover styles.
* Move Notes popover icon to be next to name
* For innate, make name be bold and centered when starting with dashes.

## Screenies

| Old | New |
| ---- | ---- |
| image | image |

Others:
image

image
image

Button examples for spells with or without notes.
image

### CODE REVIEW

File: bin/main.html [hunk 1]
```diff
@@ -939,27 +939,16 @@ <h2>
...
-      <input class="spellcasttime-grid" name="attr_spellcasttime" type="text" />
-      <select class="spelltype-grid" name="attr_spelltype" type="text">
-        <option value="-">None</option>
-        <option value="touch">Touch</option>
-        <option value="ray">Ray</option>
-        <option value="missile">Missile</option>
-      </select>
-      <input class="spellrange-grid" name="attr_spellrange" type="text" />
-      <input class="spelltarget-grid" name="attr_spelltarget" type="text" />
-      <input class="spellduration-grid" name="attr_spellduration" type="text" />
```
Comment (jazeee) [inline bin/main.html:962]: I rearranged to move the popover next to the name.

File: ui/basicTab/basicTab.pug [hunk 2]
```diff
@@ -2,7 +2,7 @@ include ../global/advantages.pug
 
 // Basic section 
 .basic.section.basic-tab
-  .advantages-and-feats.horizontal-flex
+  .advantages-and-feats
```
Comment (jazeee) [inline ui/basicTab/basicTab.pug:5]: As a project grows larger, it's generally better to name classes by concept, not by style, and style them specifically.

File: ui/buttons/buttons.scss [hunk 3]
```diff
@@ -12,9 +12,18 @@ button.actionRollButton {
   border-radius: 4px;
 }
 
+@mixin hovered-button($background-color: $button-roll-color-2) {
+  &:hover {
+    // Remove Roll20 default styles
+    background-image: unset;
+    background-color: $background-color;
+  }
+}
```
Comment (jazeee) [inline ui/buttons/buttons.scss:21]: Fixes the white hover on buttons, and cleans up.

**Comment by johnlamping** [comment]
Nice.

One small thing: the attribute column should also be fixed width. It just needs to be wide enough to hold "IQ+DX".

Two larger things, maybe for new issues:
   Using the initial "-" to change formatting doesn't make it obvious to the user that they have that option. It might be better to add a column, like the Discipline? column, that says whether a row is the basic innate casting ability for a sphere. It could be titled something like "Sphere?", and have two values, "S", or " ".
   One disadvantage of the note popovers is that there is no way to tell if there is a note, without clicking to see. It would be nice if the styling of the magnifying class changed if there was a note to be found there.

**Comment by jazeee** [comment]
Fixed the attribute width.
image

Updated to add a `S` option and style based on that.

image

I'll work on the note button as a separate ticket. I also felt it needs a bit more.
https://github.com/epic-power-rpg/roll20/issues/67

**Comment by jazeee** [comment]
Updated.

**Comment by jazeee** [comment]
@johnlamping feel free to review last commit that adds the styling changes and `Sphere?` column.

I also added a tooltip for this long column:
image

**Comment by jazeee** [comment]
Also reworked the notes button to look like:
image

**Review by johnlamping — APPROVED** [review]
👍!

**Pull request merged**